str/helper/merge_str_vcf_combiner.py

```diff
@@ -100,7 +46,7 @@ def main(input_dir, output):
     temporary_gt_file = 'temporary_gt_file.txt'
     with open(temporary_gt_file, 'w', encoding='utf-8') as handle:
         # Process each input file
-        for key in SORTED_KEY_ORDER:
+        for key in sorted(list(input_files_dict.keys())):
```

Comment:
```suggestion
        for key in sorted(input_files_dict.keys()):
```
Just a nit, but sorting any collection will return a list